pkg/placement/main.go

```diff
@@ -71,61 +54,34 @@ func NewPlacementTranslator(
 	numThreads int,
 	ctx context.Context,
 	locationPreInformer edgev1a1informers.LocationInformer,
-	// pre-informer on all SinglePlacementSlice objects, cross-workspace
+	// pre-informer on SinglePlacementSlice objects
 	epPreInformer edgev1a1informers.EdgePlacementInformer,
-	// pre-informer on all SinglePlacementSlice objects, cross-workspace
+	// pre-informer on SinglePlacementSlice objects
 	spsPreInformer edgev1a1informers.SinglePlacementSliceInformer,
-	// pre-informer on syncer config objects, should be in mailbox workspaces
+	// pre-informer on syncer config objects, should be in mailbox spaces
 	syncfgPreInformer edgev1a1informers.SyncerConfigInformer,
 
-	customizerPreInformer edgev1a1informers.CustomizerInformer,
-	// pre-informer on Workspaces objects
-	mbwsPreInformer tenancyv1a1informers.WorkspaceInformer,
-	// all-cluster clientset for kcp APIs,
-	// needed to manipulate TMC objects in mailbox workspaces
-	kcpClusterClientset kcpclusterclientset.ClusterInterface,
-	// needed for enumerating resources in workload mgmt workspaces
-	discoveryClusterClient clusterdiscovery.DiscoveryClusterInterface,
-	// needed to watch for new resources appearing
-	crdClusterPreInformer apiextinformers.CustomResourceDefinitionClusterInformer,
-	// needed to watch for new resources appearing
-	bindingClusterPreInformer bindinginformers.APIBindingClusterInformer,
-	// needed to read and write arbitrary objects
-	dynamicClusterClient clusterdynamic.ClusterInterface,
-	// to read and write syncer config objects
-	edgeClusterClientset edgeclusterclientset.ClusterInterface,
-	// for monitoring namespaces in mailbox workspaces
-	nsClusterPreInformer kcpkubecorev1informers.NamespaceClusterInformer,
-	// for creating namespaces in mailbox workspaces
-	nsClusterClient kcpkubecorev1client.NamespaceClusterInterface,
+	spaceclient msclient.KubestellarSpaceInterface,
```

Comment:
"space client" sounds like a clientset for Space objects, which this is not. I would give this field a more distinctive name. `msclient` sounds good to me.